src/components/ftbot/TradeList.vue

```diff
@@ -20,7 +20,13 @@
       @row-selected="onRowSelected"
     >
       <template #cell(actions)="row">
-        <b-button :id="`btn-actions_${row.index}`" class="btn-xs" size="sm" title="Actions">
+        <b-button
+          :id="`btn-actions_${row.index}`"
+          class="btn-xs"
+          size="sm"
+          title="Actions"
+          href="#"
```

Comment:
so the fix is adding a href="#" to the button?

i wonder how/why this works on other browsers ...

Reply:
Yeah, just found this in the docs now  too :+1: